nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const router = express.Router();
+const reviews = require("./../data/reviews.json");
+
+router.get("/", async (req, res) => {
+  try {
+    res.json(reviews);
+  } catch (error) {
+    throw error;
+  }
+});
+
+router.get("/:id", async (req, res) => {
+  try {
+    const result = getReviews(parseInt(req.params.id));
+    Object.keys(result).length === 0
+      ? res.status(404).json({ error: `${req.params.id} not found` })
+      : res.status(200).json(result);
+  } catch (error) {
+    throw error;
+  }
+});
+
+const getReview = (id) => {
```

Comment:
Same as the getMeal and getReservation, here though you're calling a plural name but defined a singular :)

Reply:
Changed it to return reviews.find((review) => review.id === id);